apps/expo/src/components/ui/button/button.tsx

```diff
@@ -54,6 +54,7 @@ const Button = ({
   rounded,
   className,
   children,
+  testID,
```

Comment:
We don't need to explicitly define this prop as it already comes in the the ...props that I spread.

Reply:
got it